instruqt-tracks/terraform-intro-gcp/tf-graph/check-workstation

```diff
@@ -7,7 +7,16 @@ if [ -f /tmp/skip-check ]; then
     exit 0
 fi
 
-# Fix this after getting the solve script working.
-curl -I http://localhost:5000 | grep -q "HTTP/1.0 200 OK" || fail-message "Oops, it looks like the Blast Radius tool is not running."
+# Make sure user actually started blast-radius
+BR_PID=$(lsof -i :5000 -F p |awk '/^p/{print substr($1,2);}')
+if [ -z "$BR_PID" ]; then
+    fail-message "Looks like Blast Radius is not running, please start it"
+else
+    # And now shut it down, since we rely on it *not* being running
+    # later on, and thanks to IL-710 and related, Instruqt doesn't
+    # reliably shut down processes when you move from one challenge
+    # to another
+    kill ${BR_PID}
```

Comment:
We should probably do things like this in the setup script of the next challenge just as a matter of practice.  If we do them in check scripts then the user can't use what we stopped after running the check script.  While that isn't a practical issue most of the time, and particularly not with this script in this challenge, it's not outside the realm of possibility that we might stop something the user needs to fix whatever we failed later on in the script.

Reply:
Fair enough, and good point, especially if we have more than one thing to ensure isn't running (i.e. we have two things to stop, user runs check, first thing is clean so we stop it but second isn't Check fails, but now first thing, which may be necessary to complete the challenge, is no longer running). And in fact, Instruqt has `cleanup-*` scripts, which "A challenge cleanup script runs after the learner completes a challenge before Instruqt sets up the next challenge."